src/components/OnlineChecker/OnlineChecker.js

```diff
@@ -0,0 +1,88 @@
+//@flow
+/*
+  This component checks to see if the the user is online or not. 
+  Failing to be online should feedback to the user as opposed to leaving them hanging.
+*/
+import React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { Z_INDICES, COLORS } from '../../constants';
+
+import * as actions from '../../actions';
+import { getOnlineState } from '../../reducers/app-status.reducer';
+
+import type { Dispatch } from '../../actions/types';
+
+const InfoBar = styled.div`
```

Comment:
Please move `InfoBar` and `SVG` down and add it before `mapStateToProps` as we're having styled-components before the redux part at the end of the file.

![grafik](https://user-images.githubusercontent.com/3046542/54396403-1fe98a00-46b3-11e9-987a-357ee84281d3.png)
I think the `InfoBar` would look better with a transparent white `background-color: ${COLORS.transparentWhite[100]};` as in the screenshot.